frontend/src/api/api.ts

```diff
@@ -1,4 +1,6 @@
 import axios from 'axios';
-import { User } from '.';
+import { RegisterUserRequest, User } from '.';
 
-export const fetchAllUsers = () => axios.get<{}, { data: User[] }>('/api/users');
+export const fetchAllUsers = () => axios.get<unknown, { data: User[] }>('/api/users');
+
+export const registerUser = (request: RegisterUserRequest) => axios.post('/api/users', request)
```

Comment:
The new endpoint should be /api/signup after secrutiy branch merge if I remember correctly.